site/src/component/Review/Review.scss

```diff
@@ -26,22 +26,46 @@
   justify-self: flex-end;
   margin-right: 3vw;
 }
-.rating-label {
-  line-height: 1rem;
-  font-size: 1rem;
-}
 .rating {
   margin: 1vh 0;
   border-radius: var(--border-radius);
   width: 5vw;
   height: 5vw;
```

Comment:
Change this to use a fixed width instead of `vw`

Reply:
Having a fixed height also makes it so you only have to check a couple different screen sizes (i.e. phone, desktop) instead of checking the whole continuous range